community/urlview/sources

```diff
@@ -0,0 +1,2 @@
+http://ftp.debian.org/debian/pool/main/u/urlview/urlview_0.9.orig.tar.gz
+http://ftp.debian.org/debian/pool/main/u/urlview/urlview_0.9-21.diff.gz
```

Comment:
Purpose of this?

Reply:
The original software hasn't been updated in nearly 20 years, but debian has been maintaining it through patches.

Reply:
No https sources?